Package.swift

```diff
@@ -0,0 +1,18 @@
+// swift-tools-version:4.2
+import PackageDescription
+
+let package = Package(
+    name: "DropDown",
+    // platforms: [.iOS("8.0"), .macOS("10.10"), .tvOS("9.0"), .watchOS("2.0")],
```

Comment:
looks like they only support iOS, you can make this line `platforms: [.iOS(v8)]`